kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/instance/DocumentInstanceList.java

```diff
@@ -29,7 +29,7 @@
 @XmlRootElement(name = "document-instance-list")
 public class DocumentInstanceList implements ItemList<DocumentInstance> {
 
-    @XmlElement(name="document-instance")
+    @XmlElement(name="document-instances")
```

Comment:
because it represents a list/array so it better matches the wrapper when marshalled